packages/css/native/shared/Css_AtomicTypes.ml

```diff
@@ -2038,8 +2038,8 @@ module Filter = struct
   let toString x =
```

Comment:
Oh, we have Filter.toString in AtomicTypes. Can you use Filter.toString in Css_Js_Core, Css_Legacy_Core instead of the same implementation of the functions?

Reply:
Right, good spot!